### ISSUE TITLE
SVGLayers renderer is never advertised and is not checked in handlers

### ISSUE THREAD

**Issue by JRegimbal** [Shared-Reality-Lab/IMAGE-server#542]
https://github.com/Shared-Reality-Lab/IMAGE-server/blob/00b05ff059a3c8f9fc2c19f074c50eb76cb604e1/handlers/svg-object-detection/od_svg.py#L62

Currently, only the DebugMode capability is sent by the client (see https://github.com/Shared-Reality-Lab/IMAGE-browser/blob/39a38c7e65a9084e4ebaca367c591c465df0cac2/src/utils.ts#L63) and checked by the SVG debug handlers (ex above).
This is not correct behavior. The client MUST advertise all renderers it will display in the `renderers` field of a request. Similarly, handlers MUST NOT return renderings in a format that was not advertised by the client.

So far, this has only been checked in PRs. Considering this got through though, it may be necessary to also enforce this behavior at runtime (#541).

Assigning to @rohanakut and @jaydeepsingh25 for the handler and extension sides respectively and since this will require coordination.

**Comment by jeffbl** [comment]
@JRegimbal I agree that the extension should be advertising the renderer correctly. Question for you to make sure clear: we don't want the renderings to be produced unless the debug flag is turned on, since we don't want these appearing unless in debug mode. (And thus, it doesn't make sense to create the renderings on the server!) So this would mean that although the extension should always advertise that it can render the SVGs, the handlers for creating the debug SVGs will still look at the debug flag (as currently implemented), and if not set, will not create renderings. Any issues you see what that approach?

**Comment by JRegimbal** [comment]
@jaydeepsingh25 why are these two different renderers?
